server/main.py

```diff
@@ -168,4 +168,4 @@ def hello():
 
 if __name__ == "__main__":
     app.debug = True
-    app.run(host=sys.argv[1], port=int(sys.argv[2]))
+    app.run(host='127.0.0.1', port=5000)
```

Comment:
This changes the functionality of the script to not accept the port and ip as command-line arguments. Is this correct?